src/app/components/trade/SwapPanel.tsx

```diff
@@ -256,6 +258,9 @@ export default function SwapPanel() {
               onCurrencySelect={handleInputSelect}
               id="swap-currency-input"
               currencyList={CURRENCY}
+              onPercentSelect={handleInputPercentSelect}
+              percent={percents[Field.INPUT]}
+              isLoggedIn={!!account}
```

Comment:
You don't need to make one more field. And `isLoggedIn` is useless for the component. So remove `isLoggedIn` prop.
`onPercentSelect={account && handleInputPercentSelect}`